src/battle_script_commands.c

```diff
@@ -3136,15 +3136,15 @@ void SetMoveEffect(bool32 primary, u32 certain)
                         gBattlerAbility = gEffectBattler;
                         RecordAbilityBattle(gEffectBattler, ABILITY_INNER_FOCUS);
                         gBattlescriptCurrInstr = BattleScript_FlinchPrevention;
-                    } else 
+                    } else
```

Comment:
```suggestion
                    }
                    else
```
I know it's not your fault, but might as well clean this up now that we see it, haha